### DISCUSSION TITLE
Welcome to uncletopia Discussions!

### DISCUSSION THREAD

**Discussion by leighmacdonald** [leighmacdonald/uncletopia#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by acrostle** [comment]
why am i banned for "racial slurs" i haven't said any

**Comment by Jeliciousz** [comment]
How does Uncletopia have class limits while not having mp_tournament mode enabled?

**Reply by leighmacdonald** [reply]
We use this plugin https://github.com/leighmacdonald/uncletopia/blob/master/roles/sourcemod/files/addons/sourcemod/scripting/classrestrict.sp

**Reply by Jeliciousz** [reply]
Thank you!

**Reply by Jeliciousz** [reply]
How can I get the UI (showing the class limit/whether the specific class is full or not) to show on the class select menu like what shows up with mp_tournament's class limits?

**Comment by Jeliciousz** [comment]
It's very hard to find resources for sourcemod online, so sorry if this isn't really the place to ask, but...
When I start my server, stopwatch.smx and EdictLimiter.smx both complain about not being able to create DHOOks / detours for CTFGameRules::ManageStopwatchTimer and CreateFeignDeathRagdoll respectively. Is this normal? Is it safe to ignore? Or has something gone wrong?

**Reply by Jeliciousz** [reply]
Also, what about tf2-comp-fixes saying "Support for the server's operating system has been deprecated, some features are disabled"
I'm running the server on Windows 10
